contracts/hardhat/contracts/RewardPoolV2.sol

```diff
@@ -130,6 +231,34 @@ contract RewardPoolV2 is ERC20WrapperUpgradeable, OwnableUpgradeable {
         return true;
     }
 
+    /**
+     * @dev Calculates the amount of underlying tokens that can be redeemed for a given staking token value.
+     */
+    function calculateWithdrawalAmount(
+        uint256 redeemAmount
+    ) public view returns (uint128) {
+        return uint128(redeemAmount.mul(exchangeRate()));
+    }
+
+    /**
+     * @dev Retrieves the actual exchange rate of the staking token to underlying token.
+     */
+    function exchangeRate() public view returns (uint128) {
+        uint128 previousExchangeRate = _exchangeRate > 0
+            ? _exchangeRate
+            : uint128(FixedMath.ONE);
```

Comment:
Propose to use `uint128` in FixedMath library to avoid many type conversions (256 -> uint128), be cause of almost variables are of uint128